### PR TITLE
Move due date from index to request pages

### PR DESCRIPTION
**Pull request by allbecauseyoutoldmeso**
### What

- Remove the hard-coded '15 days' due date from the requests index page.
- Show the due date for each request on the edit request page.
- Move some strings to locale files.

Linked to https://github.com/unboxed/bops/pull/672 (ensures correct due date is pulled through for description change requests).

### Story card

https://trello.com/c/O1qA5sVv/682-support-both-15-day-validation-and-5-day-post-validation-time-limited-request-from-applicants-3

### Screenshots
Screenshot 2022-06-23 at 16 40 01
Screenshot 2022-06-23 at 16 40 08
Screenshot 2022-06-23 at 16 40 17
Screenshot 2022-06-23 at 16 40 27
Screenshot 2022-06-23 at 16 40 39
Screenshot 2022-06-23 at 16 40 51

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -26,6 +26,7 @@ group :development, :test do
   gem "byebug", platforms: %i[mri mingw x64_mingw]
   gem "capybara"
   gem "factory_bot_rails"
+  gem "pry"
```
Comment (allbecauseyoutoldmeso) [inline Gemfile:29]: I know we've got byebug already, but I think it's nice to have both as lots of developers seem pretty evenly split between preferring one or the other!

Comment (allbecauseyoutoldmeso) [inline Gemfile:29]: I marginally prefer this to byebug and I think some devs do and some devs don't, so might make sense to have both?

File: spec/system/additional_document_validation_request_spec.rb [hunk 2]
```diff
@@ -18,6 +18,7 @@
           "state": "open",
           "document_request_type": "Roman theatre plan",
           "document_request_reason": "I do not see a vomitorium in this.",
```
Comment (allbecauseyoutoldmeso) [inline spec/system/additional_document_validation_request_spec.rb:20]: 🤣 🤷

File: Gemfile (hunk 1 above)
Comment (benbaumann95) [reply Gemfile:29]: Yeah I prefer pry too.. I think we can just have the one gem to cover both though?
We've used this before in the past https://github.com/deivid-rodriguez/pry-byebug

**Review by benbaumann95 — APPROVED** [review]
Looks great, nice job with moving the text over to the locale 🎉 🙏

File: Gemfile (hunk 1 above)
Comment (allbecauseyoutoldmeso) [reply Gemfile:29]: I'm not sure it's being given much TLC...last update a year ago and they've not been merging dependabot PRs, including security fixes 🤔

**Pull request merged**